charts/newrelic-k8s-metrics-adapter/values.yaml

```
@@ -11,24 +11,32 @@
 #   cluster:
 #   nrStaging:
 
+# -- -Required- New Relic [Personal API Key](https://docs.newrelic.com/docs/apis/intro-apis/new-relic-api-keys/#user-api-key) (stored in a secret). Used to connect to NerdGraph in order to fetch the configured metrics.
 personalAPIKey:
 
+# -- Enable metrics adapter verbose logs.
 verboseLog: false
 
 config:
-  # accountID is a required parameter to install the adapter.
-  # https://docs.newrelic.com/docs/accounts/accounts-billing/account-structure/account-id/
+  # -- -Required- New Relic [Account ID](https://docs.newrelic.com/docs/accounts/accounts-billing/account-structure/account-id/) where the configured metrics are stored.
```

Since these comments are rendered as markdown, could we take advantage of it to print this `-Required-` a bit more enriched?
For example
```suggestion
  # -- New Relic [Account ID](https://docs.newrelic.com/docs/accounts/accounts-billing/account-structure/account-id/) where the configured metrics are stored. (**Required**)
```

---

Done